fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -55,6 +55,151 @@ object ImageCutout {
 
 sealed trait FaciaContent
 
+object FaciaContent {
+
+  def fold[T](fc: FaciaContent)(c: (CuratedContent) => T, scc: (SupportingCuratedContent) => T,
+              ls: (LinkSnap) => T, las: (LatestSnap) => T): T = fc match {
```

Comment:
I know it's already merged, but I'd recommend giving things more descriptive names where possible.  We usually have autocomplete anyway, and it helps my poor brain =)